### ISSUE TITLE
[Bug] Drop-in assets not working on macOS client (3.8.1)

### ISSUE THREAD

**Issue by gorgatron1** [syd711/vpin-studio#570]
**Describe the bug**
drop-in assets not working on macOS client (3.8.1) -- never sees files added

**To Reproduce**

On launch I see this:

```
28-10 18:19:46.377 INFO  [Folder Monitoring Thread for "/Users/dkoski/Downloads/pinball"] d.m.v.c.u.FolderMonitoringThread - Launched Folder Monitoring Thread for "/Users/dkoski/Downloads/pinball"
28-10 18:19:46.379 INFO  [Folder Monitoring Thread for "/Users/dkoski/Downloads/pinball"] d.m.v.c.u.FolderMonitoringThread - Folder monitor failed: Modifier not supported
java.lang.UnsupportedOperationException: Modifier not supported
	at java.base/sun.nio.fs.PollingWatchService.register(Unknown Source)
	at java.base/sun.nio.fs.UnixPath.register(Unknown Source)
	at de.mephisto.vpin.commons.utils.FolderMonitoringThread.lambda$startMonitor$0(FolderMonitoringThread.java:66)
	at java.base/java.lang.Thread.run(Unknown Source)
28-10 18:19:46.379 INFO  [Folder Monitoring Thread for "/Users/dkoski/Downloads/pinball"] d.m.v.c.u.FolderMonitoringThread - Folder Monitoring Thread for "/Users/dkoski/Downloads/pinball" terminated.
28-10 18:19:46.409 INFO  [JavaFX Application Thread] d.m.v.u.d.DropInManager - Drop-in monitoring enabled: true
```

I know this was working when the drop-in assets was first introduced.

**Desktop (please complete the following information):**
 - OS: macOS 15

**Comment by syd711** [comment]
The issue here is that the recursive monitoring of a folder is only supported for Window clients. I've added a switch so that the regular folders is working again on macOS + Linux, but not subfolders.

**Comment by syd711** [comment]
Fixed with 3.9.0
